src/framework/pages/admin/resources/AbstractResourceBasePage.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * 
+ */
+package framework.pages.admin.resources;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import framework.selenium.SeleniumDriverManager;
+
+/**
+ * @author administrator
+ *
+ */
+public class AbstractResourceBasePage {
+	protected WebDriver driver;
+	protected WebDriverWait wait;
+
+	@FindBy(xpath = "(//input[@type='text'])[3]") 
+	WebElement resourceNameTextbox;
+	
+	@FindBy(xpath = "(//input[@type='text'])[4]") 
+	WebElement resourceDisplayNameTextbox;
+	
+	@FindBy(xpath = "//div[3]/div[2]/button") 
```

Comment:
Could you change this locator in that you will not use a index to locate some element